synapse/models/inet.py

```diff
@@ -396,43 +397,20 @@ def _revModl201706121318(self):
         if adds:
             self.core.addRows(adds)
 
-    @modelrev('inet', 201708141516)
-    def _revModl201708141516(self):
-        node = self.core.getTufoByProp('syn:type', 'inet:urlfile')
-        if not node:  # pragma: no cover
-            # Its possible someone deleted their syn:type=inet:urlfile node :(
-            mesg = 'No syn:type="inet:urlfile" node found during model revision.'
-            logger.warning(mesg)
-            self.core.log(logging.WARNING, mesg=mesg)
-            return
-        self.core.delRowsByIdProp(node[0], 'syn:type:names')
-        self.core.delRowsByIdProp(node[0], 'syn:type:types')
-        # Add in the new fields type if it does not exist.
-        if 'syn:type:fields' not in node[1]:
-            self.core.addRows([(node[0], 'syn:type:fields', 'url=inet:url,file=file:bytes', s_common.now())])
-
     @modelrev('inet', 201708231646)
     def _revModl201708231646(self):
-        node = self.core.getTufoByProp('syn:prop', 'inet:ipv4:type')
-        if not node:  # pragma: no cover
-            # Its possible someone deleted their syn:prop=inet:ipv4:type node :(
-            mesg = 'No syn:prop="inet:ipv4:type" node found during model revision.'
-            logger.warning(mesg)
-            self.core.log(logging.WARNING, mesg=mesg)
-            return
-        if 'syn:prop:ptype' not in node[1]:
-            self.core.addRows([(node[0], 'syn:prop:ptype', 'str', s_common.now())])
+        pass # for legacy/backward compat
```

Comment:
Why are we keeping this one but not the one above it?

Reply:
because we need something that gets us "past" the previous ( now un-needed ) version callbacks for backward compat.